sof_wrapper/config.py

```diff
@@ -5,6 +5,7 @@
 import os
 import redis
 
+DEBUG_OUTPUT_DIR = os.getenv("DEBUG_OUTPUT_DIR", '/tmp')
```

Comment:
@ivan-c Would it make sense to create a volume or bind-mount and define a variable to make it easier to pull the debugging output from the docker container?

Reply:
Bind-mount makes the most sense (feel free to create a PR in cosri-environments), but `docker cp` should be good enough if we're doing this only occasionally